sdk/communication/azure-communication-callingserver/src/test/java/com/azure/communication/callingserver/CallConnectionAsyncLiveTests.java

```diff
@@ -27,15 +27,10 @@
 
     @ParameterizedTest
     @MethodSource("com.azure.core.test.TestBase#getHttpClients")
+    @DisabledIfEnvironmentVariable(
+        named = "COMMUNICATION_SKIP_INT_CALLINGSERVER_TEST",
```

Comment:
Maybe a better name. This is not for INT only.

Reply:
INT was used for **INT environment** not **Integration tests.** :-)

Reply:
env variable renamed to RUN_CALLINGSERVER_TEST_RECORD